lib/uniswapV2SubGraph.ts

```diff
@@ -32,3 +46,31 @@ export async function getEthPrice() {
 
   return 1;
 }
+
+export async function getPairAddress(_address) {
+  let c = { 
+    coin: "", 
+    logo: "",
+    decimals: 18
+  };
+  const gQL = pairQL(_address);
+
+  for (let i in urlArr) {
+    let res = await fetch(urlArr[i], {
+      method: 'POST',
+      headers: {
+        'Accept': 'api_version=2',
+        'Content-Type': 'application/graphql'
+      },
+      body: JSON.stringify({ query : gQL })
+    });
+
+    let { data: { pair } } = await res.json();
+    if(pair != null){
+      c.coin = `${pair.token0.symbol}-${pair.token1.symbol}`
```

Comment:
what about making it an array so it's easier and flexible to handle it from the components?

Reply:
if multiple then can use array otherwise object